### PR TITLE
Feat/update onboarding

### PR DESCRIPTION
**Pull request by ZzRanger**
Update onboarding

### CODE REVIEW

File: src/components/home/Profile.tsx [hunk 1]
```diff
@@ -26,8 +26,8 @@ export default function ProfilePage({ isDesktop }: ProfilePageProps): JSX.Elemen
     if (!isLoading) {
       setName(data?.profile?.name ?? '');
       setEmail(data?.email ?? '');
-      setMajors(data?.profile?.majors ?? []);
-      setMinors(data?.profile?.minors ?? []);
+      // setMajors(data?.profile?.majors ?? []);
```
Comment (thekevinge) [inline src/components/home/Profile.tsx:29]: I think you accidentally left comments :)

File: src/modules/common/data.ts [hunk 2]
```diff
@@ -8,10 +8,7 @@ export function generateSemesters(
   count: number,
```
Comment (thekevinge) [inline src/modules/common/data.ts:8]: Should we move this function to ``utilFunctions.ts``?

**Review by thekevinge — COMMENTED** [review]
Are we just going to use email as a user's "username" when displaying login info/on the generated PDF? Can the user have multiple accounts with the same emails from different providers?

File: src/components/home/Profile.tsx (hunk 1 above)
Comment (ZzRanger) [reply src/components/home/Profile.tsx:29]: oop thx. We prolly should change profile to just show start & end semesters instead in line w/ the new onboarding. I'm thinking I'll make that one of the tasks for next week's pair programming.

File: src/modules/common/data.ts (hunk 2 above)
Comment (ZzRanger) [reply src/modules/common/data.ts:8]: Ye I'll do that

**Comment by ZzRanger** [comment]
> Just a few questions I had:
> 
>     * Are we just going to use email as a user's "username" when displaying login info/on the generated PDF?

We can use the "name" field from profile (user.profile.name from User table or profile.name from Profile table)

>     * Can the user have multiple accounts with the same email from different providers?

I think so?

**Review by thekevinge — DISMISSED** [review]

**Review by thekevinge — APPROVED** [review]

**Pull request merged**